src/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,264 @@
+                            <div>
+                              <h3 className="sr-only">Minimum Price</h3>
+                              <input
+                                type="number"
+                                name="min-price"
+                                id="min-price"
+                                placeholder="Min"
+                                className="border border-gray-300 rounded-md px-3 py-2 w-1/3 md:w-1/4 mr-1 focus:outline-primary"
+                              />
+                              <h3 className="sr-only">Maximum Price</h3>
+                              <input
+                                type="number"
+                                name="max-price"
+                                id="max-price"
+                                placeholder="Max"
+                                className="border border-gray-300 rounded-md px-3 py-2 w-1/3 md:w-1/4 focus:outline-primary"
+                              />
+                              <button
+                                type="button"
+                                className="bg-primary text-background px-3 py-1 rounded-md ml-5"
+                              >
+                                Apply
+                              </button>
+                            </div>
+                            {section.options.map((option, optionIdx) => (
+                              <div
+                                key={option.value.min}
+                                className="flex items-center"
+                              >
+                                <input
+                                  id={`filter-${section.id}-${optionIdx}`}
+                                  name={`${section.id}[]`}
+                                  defaultValue={option.value}
+                                  type="radio"
+                                  defaultChecked={option.checked}
+                                  className="h-4 w-4 rounded border-gray-300 text-primary focus:ring-secondary"
+                                />
+                                <label
+                                  htmlFor={`filter-${section.id}-${optionIdx}`}
+                                  className="ml-3 min-w-0 flex-1 text-primary select-none"
+                                >
+                                  {option.label}
+                                </label>
+                              </div>
+                            ))}
+                          </div>
+                        </Disclosure.Panel>
+                      </>
+                    )}
+                  </Disclosure>
+                ))}
+              </form>
+
+              {/* Product grid */}
+              <div className="lg:col-span-3">
+                {/* Replace with your content */}
+                <div className="h-96 rounded-lg border-4 border-dashed border-gray-200 lg:h-full" />
+                {/* /End replace */}
```

Comment:
Since you did this, you need to inform people who are working on this specific issue to basically pull your component (which is more like a page), and work on here.

Reply:
Ok no worries, i think viyan works on the order issue, i will inform her

Reply:
@Iseluin do you want me to make it standalone component?

Reply:
I think that kind of makes more sense, because you did not only built the filter section, you also built the item render section. If you wanna take the lead of this page and work with others on it, helping them, pair programming with them etc, you can use this, otherwise, I am seeing 3 components here not one 😄

Reply:
Ok I'll change and make it standalone component